src/vm/analysis/arithmetic_checker/tests.rs

```diff
@@ -69,7 +69,7 @@ fn test_bad_defines() {
 
     for (contract, error) in tests.iter() {
         assert_eq!(
-            arithmetic_check(contract),
+            arithmetic_check(contract, ClarityVersion::Clarity1),
```

Comment:
Can you actually test both Clarity versions here? I don't want to lose coverage for `Clarity2`.

Reply:
I made clarity1 and clarity2 versions of this. In general this is a good idea, but I'm not sure it's that needed in this particular instance, but can't really hurt.